public/app/features/auth-config/ProviderConfigForm.tsx

```diff
@@ -114,95 +138,106 @@ export const ProviderConfigForm = ({ config, provider, isLoading }: ProviderConf
+          onDiscard={() => {
+            reportInteraction('grafana_authentication_ssosettings_abandoned', {
+              provider,
+            });
+            reset();
+          }}
+        />
+        <Field label="Enabled" hidden={true}>
+          <Switch {...register('enabled')} id="enabled" label={'Enabled'} />
+        </Field>
+        {sections ? (
+          <Stack gap={2} direction={'column'}>
+            {sections
+              .filter((section) => !section.hidden)
+              .map((section, index) => {
                 return (
-                  <FieldRenderer
-                    key={field}
-                    field={field}
-                    control={control}
-                    errors={errors}
-                    setValue={setValue}
-                    register={register}
-                    watch={watch}
-                    unregister={unregister}
-                    provider={provider}
-                    secretConfigured={!!config?.settings.clientSecret}
-                  />
+                  <CollapsableSection label={section.name} isOpen={index === 0} key={section.name}>
+                    {section.fields
+                      .filter((field) => (typeof field !== 'string' ? !field.hidden : true))
+                      .map((field) => {
+                        return (
+                          <FieldRenderer
+                            key={typeof field === 'string' ? field : field.name}
+                            field={field}
+                            control={control}
+                            errors={errors}
+                            setValue={setValue}
+                            register={register}
+                            watch={watch}
+                            unregister={unregister}
+                            provider={provider}
+                            secretConfigured={!!config?.settings.clientSecret}
+                          />
+                        );
+                      })}
+                  </CollapsableSection>
                 );
               })}
-            </>
-          )}
-          <Box display={'flex'} gap={2} marginTop={6}>
-            <Field>
-              <Button type={'submit'} disabled={isSaving}>
+          </Stack>
+        ) : (
+          <>
+            {providerFields.map((field) => {
+              return (
+                <FieldRenderer
+                  key={field}
+                  field={field}
+                  control={control}
+                  errors={errors}
+                  setValue={setValue}
+                  register={register}
+                  watch={watch}
+                  unregister={unregister}
+                  provider={provider}
+                  secretConfigured={!!config?.settings.clientSecret}
+                />
+              );
+            })}
+          </>
+        )}
+        <Box display={'flex'} gap={2} marginTop={6}>
```

Comment:
```suggestion
        <Box display={'flex'} gap={2} marginTop={5}>
```
The margin is 40px in the designs, which is spacing 5.